### PR TITLE
remove beta note

### LINKED ISSUE #8230: Remove mount beta notification from docs
Remove preview note:
lakeFS DocumentationlakeFS Documentation
[Mount](https://docs.lakefs.io/reference/mount.html)
This section covers the Everest feature for mounting a lakeFS path to your local filesystem.

### PR DESCRIPTION
**Pull request by nadavsteindler**
Fix #8230 
Remove the note in docs that says mount is in beta

### CODE REVIEW

File: docs/reference/mount.md [hunk 1]
```diff
@@ -21,10 +21,6 @@ Everest is a complementary binary to lakeFS that allows users to virtually mount
 Once mounted, users can access the data as if it resides on their local filesystem, using any tool, library, or framework that reads from a local filesystem.
 This functionality is currently in limited support and is a Read-Only file system and is pointing to a commit in lakeFS.
 
-**Note**
-⚠️ Everest is currently in beta. There is no installation required, please [contact us](http://info.lakefs.io/thanks-lakefs-mounts) to get access to the Everest binary.
```
Comment (Isan-Rivkin) [inline docs/reference/mount.md:25]: > There is no installation required, please [contact us](http://info.lakefs.io/thanks-lakefs-mounts) to get access to the Everest binary.

Remove the `beta` but keep the rest

**Review by Isan-Rivkin — APPROVED** [review]
don't want to block, added a single change @nadavsteindler

File: docs/reference/mount.md (hunk 1 above)
Comment (nadavsteindler) [reply docs/reference/mount.md:25]: OK how is this?

**Review by Isan-Rivkin — APPROVED** [review]

File: docs/reference/mount.md (hunk 1 above)
Comment (Isan-Rivkin) [reply docs/reference/mount.md:25]: wonderful

**Review by Isan-Rivkin — APPROVED** [review]

**Pull request merged**